docs/Containers/Homebridge.md

```diff
@@ -0,0 +1,45 @@
+# Homebridge
+
+## References 
+
+* [GitHub home](https://github.com/oznu/docker-homebridge)
+* [Configuration Guide](https://github.com/oznu/docker-homebridge/wiki/Homebridge-on-Raspberry-Pi)
+* [DockerHub](https://hub.docker.com/r/oznu/homebridge)
+
+## Configuration
+
+Homebridge documentation has a comprehensive [configuration guide](https://github.com/oznu/docker-homebridge/wiki/Homebridge-on-Raspberry-Pi) which you are encouraged to read.
+
+Homebridge is configured using environment variables. In IOTstack:
+
+* If you are running new menu (master branch, the default), environment variables are kept inline in `docker-compose.yml`.
+* If you are running old menu (old-menu branch), environment variables are at the path:
+
+	```
+	~/IOTstack/services/homebridge/homebridge.env
+	```
+
+In either case, you apply changes by editing the relevant file (`docker-compose.yml` or `homebridge.env`) and then:
+
+```bash
```

Comment:
In regards to #547, could you change this from `bash` to `console`?

Reply:
Done.